Test failure: `TestIbaProbes `

```
=== RUN   TestIbaProbes/5.1.0_i-0a36daf0e2501f5e7/specific_interface_flapping
=== PAUSE TestIbaProbes/5.1.0_i-0a36daf0e2501f5e7/specific_interface_flapping
=== CONT  TestIbaProbes/5.1.0_i-0a36daf0e2501f5e7/specific_interface_flapping
    api_iba_probes_test.go:384: Get Predefined Probe By Name specific_interface_flapping
    api_iba_probes_test.go:388: Instantiating Probe specific_interface_flapping
    api_iba_probes_test.go:395: 
        	Error Trace:	/Users/cmarget/projects/work/apstra-go-sdk/apstra/api_iba_probes_test.go:395
        	Error:      	An error is expected but got nil.
        	Test:       	TestIbaProbes/5.1.0_i-0a36daf0e2501f5e7/specific_interface_flapping
--- FAIL: TestIbaProbes/5.1.0_i-0a36daf0e2501f5e7/specific_interface_flapping (0.61s)
```

--- a/apstra/api_iba_probes_test.go
+++ b/apstra/api_iba_probes_test.go
@@ -14,6 +14,7 @@ import (
 	"log"
 	"testing"
 
+	"github.com/hashicorp/go-version"
 	"github.com/stretchr/testify/require"
 )
 
@@ -369,10 +370,12 @@ func TestIbaProbes(t *testing.T) {
 				"eastwest_traffic":                   true,
 				"vxlan_floodlist":                    true,
 				"fabric_hotcold_ifcounter":           true,
-				"specific_interface_flapping":        true,
 				"evpn_vxlan_type3":                   true,
 				"specific_hotcold_ifcounter":         true,
 				"spine_superspine_hotcold_ifcounter": true,
+			}
+			if version.MustConstraints(version.NewConstraint("<5.1.0")).Check(client.client.apiVersion) {
+				expectedToFail["specific_interface_flapping"] = true
 			}
 
 			for _, predefinedProbe := range predefinedProbes {

